src/client/datascience/interactive-window/nativeInteractiveWindow.ts

```diff
@@ -289,10 +289,12 @@ export class NativeInteractiveWindow implements IInteractiveWindowLoadable {
         // Add message to the notebook document
         const edit = new WorkspaceEdit();
         const notebookDocument = this.notebookDocument;
+        const markdownCell = new NotebookCellData(NotebookCellKind.Markup, message, MARKDOWN_LANGUAGE);
+        markdownCell.metadata = { isInteractiveWindowMessageCell: true };
```

Comment:
Make sure messages like "3 cells were cancelled due to an error..." aren't included in exported interactive window history